CMakeLists.txt

```diff
@@ -177,7 +177,7 @@ include(DaemonPlatform)
 ################################################################################
 
 # Dependencies version, this must match the number in external_deps/build.sh
-set(DEPS_VERSION 6)
```

Comment:
You have another outstanding pull request changing the deps package, so maybe you could merge only the patch file stuff in this PR and move the CMake changes to #713.

Reply:
I opened that other PR to have a different thread to talk about things, but now that everything looks ready to merge, I imported all the PRs here to merge all of them in one go so we only have one new package to send per architecture.